scipy/stats/tests/test_stats.py

```diff
@@ -4117,8 +4117,11 @@ def test_some_code_paths(self):
         _compute_outer_prob_inside_method(1, 1, 1, 1)
         _count_paths_outside_method(1000, 1, 1, 1001)
 
-        assert_raises(FloatingPointError, _count_paths_outside_method, 1100, 1099, 1, 1)
-        assert_raises(FloatingPointError, _count_paths_outside_method, 2000, 1000, 1, 1)
+        with np.errstate(invalid='raise'):
+            assert_raises(FloatingPointError, _count_paths_outside_method,
+                          1100, 1099, 1, 1)
+            assert_raises(FloatingPointError, _count_paths_outside_method,
+                          2000, 1000, 1, 1)
```

Comment:
If you want to make sure that the `invalid` `RuntimeWarning` warning is raised.
```suggestion
        message = "invalid value encountered"
        with pytest.warns(RuntimeWarning, match=message):
            _count_paths_outside_method(1100, 1099, 1, 1)
        message = "invalid value encountered"
        with pytest.warns(RuntimeWarning, match=message):
            _count_paths_outside_method(2000, 1000, 1, 1)
```
Or, you could make sure that this leads to the warning `ks_2samp: Exact calculation unsuccessful. Switching to method=asymp.` when we attempt to use `ks_2samp` on samples of these sizes.